test/test_config.rb

```diff
@@ -58,4 +54,8 @@ def check_context(context, ssl: true)
       assert_equal(OpenSSL::SSL::VERIFY_NONE, context.ssl_options[:verify_ssl])
     end
   end
+
+  def config_file(name)
+    File.new(File.join(File.dirname(__FILE__), 'config', name))
+  end
```

Comment:
things that start with `test_` are tests, so I found this confusing/dangerous ... so changing to config_file